onnx/defs/math/old.cc

```diff
@@ -5,7 +5,7 @@
 #include "onnx/defs/schema.h"
 
 namespace ONNX_NAMESPACE {
-
+using namespace Common;
```

Comment:
I think `using Common::ONNX_DOMAIN;` should be enough in most of the cases.

Reply:
Common namespace is not used for now.